localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+c_stonesplinter_valley_adj;Stonesplinter;Brisepierre;;;Rompecantos;;;;;;;;;x
+b_stonesplinter_valley;Stonesplinter;Brisepierre;;;Rompecantos;;;;;;;;;x
+b_stonesplinter_valley_adj;Stonesplinter;Brisepierre;;;Rompecantos;;;;;;;;;x
+PROV160;Stonewrought;Formepierre;;;Tres Cabezas;;;;;;;;;x
+c_stonewrought_dam;Stonewrought;Formepierre;;;Tres Cabezas;;;;;;;;;x
+c_stonewrought_dam_adj;Stonewrought;Formepierre;;;Tres Cabezas;;;;;;;;;x
+b_stonewrought_dam;Stonewrought;Formepierre;;;Tres Cabezas;;;;;;;;;x
+b_stonewrought_dam_adj;Stonewrought;Formepierre;;;Tres Cabezas;;;;;;;;;x
+d_loch_modan;Thelsamar;Thelsamar;;;Thelsamar;;;;;;;;;x
+d_loch_modan_adj;Thelsamar;Thelsamar;;;Thelsamar;;;;;;;;;x
 PROV158;Kren'sen;;;;;;;;;;;;;x
 c_hidden_ravine;Kren'sen;;;;;;;;;;;;;x
 c_hidden_ravine_adj;Kren'sen;;;;;;;;;;;;;x
-d_high_northeron;Thundermar;;;;;;;;;;;;;x
-d_high_northeron_adj;Thundermar;;;;;;;;;;;;;x
-PROV176;Thorvald;;;;;;;;;;;;;x
-c_thorvald_ravine;Thorvald;;;;;;;;;;;;;x
-c_thorvald_ravine_adj;Thorvald;;;;;;;;;;;;;x
-b_thorvald_ravine;Thorvald;;;;;;;;;;;;;x
-b_thorvald_ravine_adj;Thorvald;;;;;;;;;;;;;x
-d_western_arathi;Stromgarde;Stromgarde;;;;;;;;;;;;x
+d_high_northeron;Thundermar;Tonnemar;;;Bramal;;;;;;;;;x
+d_high_northeron_adj;Thundermar;Tonnemar;;;Bramal;;;;;;;;;x
+PROV176;Thorvald;Thorvald;;;Thorvald;;;;;;;;;x
+c_thorvald_ravine;Thorvald;Thorvald;;;Thorvald;;;;;;;;;x
+c_thorvald_ravine_adj;Thorvald;Thorvald;;;Thorvald;;;;;;;;;x
+b_thorvald_ravine;Thorvald;Thorvald;;;Thorvald;;;;;;;;;x
+b_thorvald_ravine_adj;Thorvald;Thorvald;;;Thorvald;;;;;;;;;x
+d_western_arathi;Stromgarde;Stromgarde;;;Stromgarde;;;;;;;;;x
 d_western_arathi_adj;Stromic;Stromgardienne;;;;;;;;;;;;x
-d_eastern_arathi;Go'shek;;;;;;;;;;;;;x
-d_eastern_arathi_adj;Go'shek;;;;;;;;;;;;;x
-d_cult_of_sunwell;The Sunwell Cult;Le Culte du Puits de soleil;Kult des Sonnenbrunnens;;;;;;;;;;;x
-d_cult_of_sunwell_adj;Sunwell;;;;;;;;;;;;;x
-d_old_ways;The Church of the True Ways;;Kirche der wahrer Wege;;;;;;;;;;;x
-d_old_ways_adj;Druidic;;druidisch;;;;;;;;;;;x
-d_wolf_cult_religion;The Wolf Cult;Le Culte du Loup;Kult des Wolfs;;;;;;;;;;;x
-d_wolf_cult_religion_adj;Wolf;;w�lfisch;;;;;;;;;;;x
-d_ursine;The Ursine Cult;Le Culte-Ours;Kult des B�ren;;;;;;;;;;;x
-d_ursine_adj;Ursine;;b�risch;;;;;;;;;;;x
-d_kaldorei_religion;The Priesthood of Elune;Le Clerg� d'�lune;Elunes Priesterschaft;;;;;;;;;;;x
-d_kaldorei_religion_adj;Elune's;;elunisch;;;;;;;;;;;x
-d_cult_of_loa;The Cult of Loa;Le Culte des Loas;Loa-Kult;;;;;;;;;;;x
+d_eastern_arathi;Go'shek;Go'shek;;;Go'shek;;;;;;;;;x
+d_eastern_arathi_adj;Go'shek;Go'shek;;;Go'shek;;;;;;;;;x
+d_cult_of_sunwell;The Sunwell Cult;Le Culte du Puits de soleil;Kult des Sonnenbrunnens;;Culto de la Fuente del Sol;;;;;;;;;x
+d_cult_of_sunwell_adj;Sunwell;Puits de soleil;sonnenbrunnens;;Fuente del Sol;;;;;;;;;x
+d_old_ways;The Church of the True Ways;Église des Vraies Voies;Kirche der wahrer Wege;;La iglesia de los Caminos Verdaderos;;;;;;;;;x

```

Comment:
Kirche der wahren Wege

Reply:
This wasn't my German translation, but I'll update it all the same. :)